vpr/src/draw/draw.cpp

```diff
@@ -1373,22 +1380,22 @@ void clear_colored_locations() {
 // This routine takes in a (x,y) location.
 // If the input loc is marked in colored_locations vector, the function will return true and the correspnding color is sent back in loc_color
 // otherwise, the function returns false (the location isn't among the highlighted locations)
-bool highlight_loc_with_specific_color(int x, int y, ezgl::color& loc_color) {
+bool highlight_loc_with_specific_color(int x, int y, int layer, ezgl::color& loc_color) {
```

Comment:
The function currently assigns the x,y,layer parameters into a t_pl_loc structure which stores x,y,layer coords. Should I just use that data structure for the input parameter of the function rather than using t_physical_tile_loc ? Not sure if there's any special use of using one over the other.

Reply:
You could pass the t_pl_loc if it's more convenient.